crates/polars-sql/src/functions.rs

```diff
@@ -173,6 +173,10 @@ pub(crate) enum PolarsSqlFunctions {
     /// ```
     Round,
 
+    // Date Functions
+    // ----
+    Date,
```

Comment:
I believe postgres _(which we are trying to mimic where possible)_ calls this [to_date](https://www.postgresql.org/docs/current/functions-formatting.html#FUNCTIONS-FORMATTING)

Reply:
Ah, I see - thanks.

So should `Date` and `date` be replaced with `ToDate` and `to_date`?

Reply:
> I believe postgres _(which we are trying to mimic where possible)_ calls this [to_date](https://www.postgresql.org/docs/current/functions-formatting.html#FUNCTIONS-FORMATTING)

Ahhh, `TO_DATE` is actually a separate function that _requires_ a second argument (a format string which is _not_ strftime, but rather one of the "YYYY-MM-DD" type ones).

In this case `DATE` is appropriate as it only needs to take the string and can be called like so:
```sql
SELECT DATE '2020-12-31' as d1, DATE('2222-2-22') as d2
```
(I just quickly checked this on a local PostgreSQL connection ;))